sky/serve/serve_utils.py

```diff
@@ -283,6 +283,22 @@ def update_service_encoded(service_name: str, version: int) -> str:
     return common_utils.encode_payload(service_msg)
 
 
+def terminate_replica(service_name: str, replica_id: int) -> None:
```

Comment:
pls return an encoded str for error message; ref https://github.com/skypilot-org/skypilot/blob/073e1898066844252b3bf8d0bf1e089b231a8ebc/sky/serve/serve_utils.py#L264-L283